main.go

```diff
@@ -26,13 +27,24 @@ var (
 )
 
 func main() {
+	flag.Parse()
+
+	switch flag.Arg(0) {
```

Comment:
Did you want to add a test for this?

It's more idiomatic to name the flag and then parse that.

Please could you follow this example -> https://gobyexample.com/command-line-flags

Reply:
I think a subcommand (instead of a flag) is more appropriate in this case.